cmd/skopeo/inspect.go

```diff
@@ -97,7 +99,13 @@ var inspectCmd = cli.Command{
 			outputData.Name = dockerImg.SourceRefFullName()
 			outputData.RepoTags, err = dockerImg.GetRepositoryTags()
 			if err != nil {
-				return fmt.Errorf("Error determining repository tags: %v", err)
+				// some registries may decide to block the "list all tags" endpoint
+				// gracefully allow the inspect to continue in this case
+				if !strings.Contains(err.Error(), "401") {
```

Comment:
@mtrmac yes.. I should have explained more. The IBM Bluemix container registry is a v2 registry that, because of original issues with a specific backend and performance blocked the "get all tags".  This restriction will probably be removed, but for now, I decided to fix the few places I knew where tools were failing when targeting a Bluemix repository.